deliver/lib/deliver/upload_screenshots.rb

```diff
@@ -243,10 +243,10 @@ def delete_screenshots(localizations, app_store_screenshot_sets_map, app_store_s
           app_store_screenshot_set.app_screenshots.each do |app_store_screenshot|
             next unless ids_to_delete.include?(app_store_screenshot.id)
 
-            UI.verbose("Deleting screenshot - #{language} #{app_store_screenshot_set.screenshot_display_type} #{app_store_screenshot.id}")
+            UI.message("Deleting screenshot - #{language} #{app_store_screenshot_set.screenshot_display_type} #{app_store_screenshot.id}")
             Deliver.retry_api_call do
               app_store_screenshot.delete!
-              UI.verbose("Deleted screenshot - #{language} #{app_store_screenshot_set.screenshot_display_type} #{app_store_screenshot.id}")
+              UI.message("Deleted screenshot - #{language} #{app_store_screenshot_set.screenshot_display_type} #{app_store_screenshot.id}")
```

Comment:
Minor: it might actually be useful to use `verbose` instead of `message`, since the bunker uses fastlane in verbose mode anyway. But maybe the idea was to use `message` on purpose so that it'd show up in the log of the bunker? (I'm not sure that works 🤔)

Reply:
Yeah, I changed it in order to show the message in the bunker logs. The `"Uploading screenshoot ..."` is already a message that appears in the logs and considering we had difficulties debugging the previous evening, I thought that being able to view this in the logs would be beneficial.

Reply:
Otherwise, if you think that it is an overkill, I'll change the `Uploading` log too to be verbose. 😄

Reply:
Btw, all of the `"Found ...", `"Removing ..."`, and `"Uploading ..."` logs are done as `UI.message()` and we can see them in the bunker logs. 😄

Reply:
Let's try with this on, if it becomes too verbose we'll disable it later 👍